infra/token-transfer-server/migrations/20190713051638-create-user.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('t3_user', {
+      email: {
+        type: Sequelize.STRING,
+        primaryKey: true,
```

Comment:
What do you think about having the primary key being an auto incrementing ID? I think email here is not immutable, and it might cause some problems with joins to the other tables (e.g. grants).

Reply:
Yes, I'll do that.

Reply:
Done